dip-0004.md

```diff
@@ -83,7 +83,7 @@ A SML entry consists of the following fields:
 | proRegTxHash | uint256 | 32 | The hash of the ProRegTx that identifies the masternode |
 | ipAddress | byte[] | 16 | IPv6 address in network byte order. Only IPv4 mapped addresses are allowed (to be extended in the future) |
 | port | uint_16 | 2 | Port (network byte order) |
-| keyIDOperator | CKeyID | 20 | The operators public key hash |
+| pubKeyOperator | BLSPubKey | 48 | The operators public key |
```

Comment:
In DIP3, keyID* and pubKeyOperator are capitalized (KeyID*, PubKeyOperator). Not sure which way is preferred, but we should make consistent since we are doing updates.

Reply:
I'd prefer camelCase (with starting lower case) everywhere.

Reply:
That works for me. I am not sure why DIP-3 is different - even in that DIP everything other than the key ID fields is camelCase already.